src/lib/logger/logger.ts

```diff
@@ -1,5 +1,10 @@
 import { Snowflake } from 'discord-api-types';
 import { Client, TextChannel } from 'discord.js';
+import stripAnsi from 'strip-ansi';
+
+const _log = console.log;
```

Comment:
These can be private variables within the logger class

Reply:
they can be, but im using a global of _log to check if replaceConsole was already run to make sure under no circumstances is it run twice. i'll rewrite the check to work differently.